accounts/migrations/0007_customuser_task_assingor_task_date_posted_task_floor_and_more.py

```diff
@@ -0,0 +1,123 @@
+                ('username', models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username')),
+                ('first_name', models.CharField(blank=True, max_length=150, verbose_name='first name')),
+                ('last_name', models.CharField(blank=True, max_length=150, verbose_name='last name')),
+                ('email', models.EmailField(blank=True, max_length=254, verbose_name='email address')),
+                ('is_staff', models.BooleanField(default=False, help_text='Designates whether the user can log into this admin site.', verbose_name='staff status')),
+                ('is_active', models.BooleanField(default=True, help_text='Designates whether this user should be treated as active. Unselect this instead of deleting accounts.', verbose_name='active')),
+                ('date_joined', models.DateTimeField(default=django.utils.timezone.now, verbose_name='date joined')),
+                ('is_admin', models.BooleanField(default=False)),
+                ('is_manager', models.BooleanField(default=False)),
+                ('room_number', models.PositiveIntegerField()),
+                ('phone_number', models.CharField(blank=True, max_length=15)),
+                ('groups', models.ManyToManyField(blank=True, related_name='customuser_set', to='auth.group', verbose_name='groups')),
+                ('user_permissions', models.ManyToManyField(blank=True, related_name='customuser_set', to='auth.permission', verbose_name='user permissions')),
+            ],
+            options={
+                'verbose_name': 'user',
+                'verbose_name_plural': 'users',
+                'abstract': False,
+            },
+            managers=[
+                ('objects', django.contrib.auth.models.UserManager()),
+            ],
+        ),
+        migrations.AddField(
+            model_name='task',
+            name='assingor',
+            field=models.CharField(default='Null', max_length=200),
+        ),
+        migrations.AddField(
+            model_name='task',
+            name='date_posted',
+            field=models.DateTimeField(default=datetime.datetime.now),
+        ),
+        migrations.AddField(
+            model_name='task',
+            name='floor',
+            field=models.PositiveIntegerField(default=0),
+        ),
+        migrations.AddField(
+            model_name='task',
+            name='priority',
+            field=models.IntegerField(choices=[(1, 'Low'), (2, 'Medium'), (3, 'High')], default=1),
+        ),
+        migrations.AlterField(
+            model_name='task',
+            name='id',
+            field=models.AutoField(primary_key=True, serialize=False),
+        ),
+        migrations.CreateModel(
```

Comment:
Not sure how to highlight multiple lines but...

The ManagerUser, RaUser and ResidentUser I'm not too sure about what they're trying to do. Are they essentially inheriting from the base CustomUser model up top? Regardless, it might be better to do a many-to-many / one-to-one relationship between the user and their associated floors or managers. Having a separate table for each manager / ra / resident seems a bit much. We could probably stick them iall n the same customuser table. This is of course assuming I'm reading this correctly.